src/gen_rules.ml

```diff
@@ -1701,7 +1845,7 @@ module Gen(P : Params) = struct
     |> Merlin.gen ~dir:ctx_dir
 
   let () = List.iter P.stanzas ~f:rules
-
+  let () = List.iter (Js_of_ocaml_rule.setup_findlib ()) ~f:add_rule
```

Comment:
This needs to be done only if we separate compilation is enabled, otherwise we are going to scan the whole findlib database for nothing on every start